flocker/acceptance/test_postgres.py

```diff
@@ -40,6 +40,12 @@
         Port(internal_port=POSTGRES_INTERNAL_PORT,
              external_port=POSTGRES_EXTERNAL_PORT),
         ]),
+    environment=frozenset({
+        'LANG': 'en_US.utf8',
+        'PGDATA': '/var/lib/postgresql/data',
+        'PG_VERSION': '9.4.1-1.pgdg70+1',
```

Comment:
Yeah, I guess that's the issue. So we need to change calculation logic so you don't need to hardcode env variables from the docker image in your config.

I guess... if there's *extra* env variables beyond ones manually configured, that should cause no side effects. And then you can just include e.g. `PATH` and `HOME`. Alternatively, if you can omit *all* env variables that were auto-added outside the config (is that visible in docker inspect at all?) that would work too.